In [1]:
import numpy as np
import scipy.stats as stats

# Task 1: State the Hypotheses
Null Hypothesis (H0): The weekly operating cost follows the theoretical cost model (W =1,000+ 
5X).
Alternative Hypothesis (H1): The weekly operating cost is higher than the theoretical cost model

In [2]:
print("Task 1: State the Hypotheses")
print("H0 (Null Hypothesis): The weekly operating cost follows the theoretical cost model (W = $1,000 + $5X).")
print("H1 (Alternative Hypothesis): The weekly operating cost is higher than the theoretical cost model.")
print()

Task 1: State the Hypotheses
H0 (Null Hypothesis): The weekly operating cost follows the theoretical cost model (W = $1,000 + $5X).
H1 (Alternative Hypothesis): The weekly operating cost is higher than the theoretical cost model.



# Task 2: Calculate the Test Statistic

In [3]:
# - Sample mean weekly cost (x̄): Rs. 3,050
sample_mean = 3050

In [4]:
# - Theoretical mean weekly cost (μ): Rs. 1,000 + 5 * 600 = Rs. 4,000
theoretical_mean = 1000 + 5 * 600  # Calculated from the given formula


In [5]:
# - Standard deviation (σ): 5 * 25 = Rs. 125
standard_deviation = 5 * 25

In [6]:
# - Sample size (n): 25
sample_size = 25

In [7]:
# Calculate the test statistic (t)
t = (sample_mean - theoretical_mean) / (standard_deviation / np.sqrt(sample_size))

In [8]:
print(f"Test Statistic (t): {t:.3f}")
print()

Test Statistic (t): -38.000



# Task 3: Determine the Critical Value

In [9]:
# Using the alpha level of 5% (α = 0.05), determine the critical value from the standard normal (Z) distribution.
alpha = 0.05

In [10]:
# Since this is a one-sided test (greater than), we use the 95th percentile.
critical_value = stats.norm.ppf(1 - alpha)

In [11]:
print(f"Critical Value: {critical_value:.3f}")
print()

Critical Value: 1.645



# Task 4: Make a Decision

In [12]:
# Compare the test statistic with the critical value to decide whether to reject the null hypothesis.
reject_null = t > critical_value

In [13]:
if reject_null:
    print("Decision: Reject the Null Hypothesis. There is strong evidence to support the restaurant owners' claim that the weekly operating costs are higher than the model suggests.")
else:
    print("Decision: Fail to reject the Null Hypothesis. There is not enough evidence to support the restaurant owners' claim that the weekly operating costs are higher than the model suggests.")
print()

Decision: Fail to reject the Null Hypothesis. There is not enough evidence to support the restaurant owners' claim that the weekly operating costs are higher than the model suggests.



# Conclusion

# There is not enough evidence to support the restaurant owners' claim that the weekly operating costs are higher than the model suggests.

# Decision: Since the test statistic (-76.071) is less than the critical value (1.645), we fail to reject the null hypothesis.